pom.xml

```diff
@@ -25,12 +25,7 @@
 		</dependency>
 
 
-		<dependency>
```

Comment:
Why are u removing it, plz explain

Reply:
it is an unused dependency which is causing build failure.